sky/resources.py

```diff
@@ -2045,6 +2163,9 @@ def add_if_not_none(key, value):
             config['volumes'] = volumes
         if self._autostop_config is not None:
             config['autostop'] = self._autostop_config.to_yaml_config()
+        if self._no_missing_accel_warnings is not None:
+            config[
+                '_no_missing_accel_warnings'] = self._no_missing_accel_warnings
```

Comment:
I think we can just do `add_if_not_none('_no_missing_accel_warnings', self._no_missing_accel_warnings)`